decidim-proposals/lib/decidim/proposals/feature.rb

```diff
@@ -37,6 +37,19 @@
     resource.template = "decidim/proposals/proposals/linked_proposals"
   end
 
+  feature.register_stat :accepted_proposals_count, primary: true do |features, start_at, end_at|
```

Comment:
@josepjaume As discussed offline with @oriolgual I'm gonna create two extra PRs after this:
- **Add homepage statistics configuration:** As an admin I will be able to decided which statistics are available in the homepage and which are highlighted.
- **Add comments stats:** I will enhance the API to add some aggregated stats so we can add the `total_comments_count` stat.